.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "shared/libwebm"]
+	path = shared/libwebm
+	url = https://github.com/webmproject/libwebm/
```

Comment:
I'm not a fan of submodules (Josh may ok it though idk). Most distros should be their own libwebm package and for the odd platform(s) that don't we should be looking into (re)creating a separate enigma-deps repo that holds other people's code. It's hard to read through this when 99% of it isn't your code.

Reply:
Josh is the one who wanted it to be a submodule, since there is no package available for it on anything besides windows.